package-overrides/github/brianreavis/selectize.js@0.12.0.json

```diff
@@ -0,0 +1,19 @@
+{
+  "directories": {
+    "lib": "dist"
+  },
+  "main": "js/selectize",
+  "registry": "jspm",
+  "dependencies": {
+    "jquery": "*",
+    "sifter": "*",
```

Comment:
I mean should we give a version range for `sifter` and `microplugin` themselves?